styleguide/derek/pattern-components/challenges-river/_challenges-river.scss

```diff
@@ -22,11 +23,42 @@
   }
 }
 
+.challenges-learnMore {
+  color: $rackspace-red;
+  display: inline-block;
+  font-size: 1em;
+  font-weight: 400;
+}
+
 .challenges-item {
   @include make-xs-column(10);
   @include make-md-column(8);
+  height: 100%;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  transition-duration: .2s;
+
+  svg {
```

Comment:
instead of using the `svg` selectors here can we use `.challenges-itemRackShape` selector instead?